src/components/Display.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+function Display (props) {
+  return <div id="display">{props.result}</div>
+}
+
+Display.defaultProps = {
```

Comment:
I acknowledge that the instruction told so to include a `defaultProp`. I'll give you that :)

Though, this is unnecessary (the instructions require some amendments) and I am kind of providing you with an ad-hoc amendment right here :) The only time we want to put a `defaultProp` on something is if the parent component may or may not pass that particular prop to the component in question (or is able to pass a `null` value for that prop). IN this case `App` (the parent of `Display`) is always passing the `result` prop to `Display`. 

In this case, however, what should have been done is put the `isRequired` property into the `result` propType below